jqModal.js

```diff
@@ -370,7 +378,8 @@
 			toTop: false,
 			onShow: onShow,
 			onHide: onHide,
-			onLoad: false
+			onLoad: false,
+			blockMsgClass: 'blockMsg'
```

Comment:
@dpesch I'd prefer not to have any side effects. lets default this to `null` and conditionally apply the $.target.parents match (https://github.com/briceburg/jqModal/pull/25/files#diff-63a527e6356eabc413f23b70fb806783R328)  if its not null.